diff --git a/org.kde.elisa.desktop.cmake b/org.kde.elisa.desktop.cmake
index 02ee7c1e..59d02b7d 100644
--- a/org.kde.elisa.desktop.cmake
+++ b/org.kde.elisa.desktop.cmake
@@ -35,7 +35,7 @@ Comment[uk]=Відтворення музики з диска і прослух
 Comment[x-test]=xxPlay local music and listen to online radioxx
 Comment[zh_CN]=播放本地音乐并收听在线广播
 Comment[zh_TW]=播放本機音樂或聆聽線上廣播
-Exec=elisa %U
+Exec=qt-runner elisa %U
 GenericName=Music Player
 GenericName[ar]=مشغّل صوتي إليسّا
 GenericName[az]=Musiqi Pleyeri
@@ -142,4 +142,8 @@ StartupWMClass=elisa
 Type=Application
 MimeType=audio/aac;audio/mp4;audio/mpeg;audio/mpegurl;audio/vnd.rn-realaudio;audio/vorbis;audio/x-flac;audio/x-mp3;audio/x-mpegurl;audio/x-ms-wma;audio/x-musepack;audio/x-oggflac;audio/x-pn-realaudio;audio/x-scpls;audio/x-speex;audio/x-vorbis;audio/x-wav;application/x-ogm-audio;audio/x-vorbis+ogg;audio/ogg;
 SingleMainWindow=true
-@ELISA_DBUSACTIVATABLE@
+X-Nemo-Single-Instance=no
+X-Nemo-Application-Type=no-invoker
+
+[X-Sailjail]
+Sandboxing=Disabled
