diff --git a/src/main.cpp b/src/main.cpp
index 995b1dbd..c2c91e0a 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -127,7 +127,7 @@ int main(int argc, char *argv[])
 #if defined Q_OS_ANDROID
     QQuickStyle::setStyle(QStringLiteral("Material"));
 #else
-    QQuickStyle::setStyle(QStringLiteral("org.kde.desktop"));
+    QQuickStyle::setStyle(QStringLiteral("org.kde.breeze"));
     QQuickStyle::setFallbackStyle(QStringLiteral("Fusion"));
 #endif
 
