diff --git a/src/main.cpp b/src/main.cpp
index 4cc3c8b8..9a0c7e07 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -150,8 +150,8 @@ int main(int argc, char *argv[])
         QQuickStyle::setStyle(QStringLiteral("org.kde.breeze"));
         QQuickStyle::setFallbackStyle(QStringLiteral("Material"));
 #else
-        QQuickStyle::setStyle(QStringLiteral("org.kde.desktop"));
-        QQuickStyle::setFallbackStyle(QStringLiteral("Fusion"));
+        //QQuickStyle::setStyle(QStringLiteral("org.kde.desktop"));
+        //QQuickStyle::setFallbackStyle(QStringLiteral("Fusion"));
 #endif
     }
 
